### PR TITLE
Added pt-PT translations

### PR DESCRIPTION
**Pull request by srtab**
Added locale folder to project and pt-PT translations in it. 
Made some missing strings available for translations too.

### CODE REVIEW

File: oauth2_provider/locale/pt/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (IvanAnishchuk) [inline oauth2_provider/locale/pt/LC_MESSAGES/django.po]: I guess these lines could use some updating. Unless you'd rather delete them completely,

File: oauth2_provider/templates/oauth2_provider/application_form.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h3 class="block-center-heading">
                     <a class="btn" href="{% block app-form-back-url %}{% url "oauth2_provider:detail" application.id %}{% endblock app-form-back-url %}">
                         {% trans "Go Back" %}
                     </a>
-                    <button type="submit" class="btn btn-primary">Save</button>
+                    <button type="submit" class="btn btn-primary">{% trans "Save" %}</button>
```
Comment (IvanAnishchuk) [inline oauth2_provider/templates/oauth2_provider/application_form.html]: Maybe a little more consistency about double/single quotes? I'm not insisting if you feel like your system is better, it's not a big deal either way.

**Comment by MattBlack85** [comment]
@n2ygk not sure how to proceed with this, this would be the first entry for translations

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 * **Backwards-incompatible** squashed migrations:
   If you are currently on a release < 1.2.0, you will need to first install 1.2.x then `manage.py migrate` before
   upgrading to >= 1.3.0.
+* Added locale folder to project.
```
Comment (n2ygk) [inline CHANGELOG.md:7]: no need to give this much detail as the locale folder is there because of the next line.

**Review by n2ygk — COMMENTED** [review]
> @n2ygk not sure how to proceed with this, this would be the first entry for translations

I'm OK with translations but we need some ground rules to make sure they are good translations and not just something pasted into Google Translate. How do other projects do this? Have a second native-language speaker review the PR?

**Comment by RainshawGao** [comment]
@n2ygk hello, when will this pr be merged? I am looking forward to this feature.

File: oauth2_provider/locale/pt/LC_MESSAGES/django.po (hunk 1 above)
Comment (n2ygk) [reply oauth2_provider/locale/pt/LC_MESSAGES/django.po:4]: I'll just remove.

**Comment by n2ygk** [comment]
Ugh, I screwed up trying to merge this from command line. See #919

**Review by n2ygk — APPROVED** [review]
Approved with my changes...

**Comment by n2ygk** [comment]
I guess @srtab didn't allow pushes so I had to make a new PR #919 for this. Closing this one.

**Pull request closed without merging**